Neos.Flow/Classes/Http/Component/SecurityEntryPointComponent.php

```diff
@@ -39,19 +40,36 @@ public function handle(ComponentContext $componentContext)
         }
 
         $actionRequest = $componentContext->getParameter(DispatchComponent::class, 'actionRequest');
-        $entryPoint = array_reduce($this->securityContext->getAuthenticationTokens(), function ($foundEntryPoint, TokenInterface $currentToken) {
-            return $foundEntryPoint ?? $currentToken->getAuthenticationEntryPoint();
-        }, null);
```

Comment:
I extracted this to a method `getFirstTokenWithEntryPoint()` that stops when the first token is found. Before this would always iterate all tokens